pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalanceProgressStats.java

```diff
@@ -234,8 +236,13 @@ public void updateOverallAndStepStatsFromLatestStepStats(
   }
 
   public static double calculatePercentageChange(int totalSegmentsToChange, int remainingSegmentsToChange) {
-    return totalSegmentsToChange == 0
-        ? 0.0 : (double) remainingSegmentsToChange / (double) totalSegmentsToChange * 100.0;
+    if (totalSegmentsToChange == 0) {
```

Comment:
oh can we not change the progress stats themselves and only have the metric emit progress done instead of remaining?